dashboard/src/features/transactions/components/New/FormActionItem.jsx

```diff
@@ -58,13 +58,13 @@ export default class ActionItem extends React.Component {
 
   render() {
     const {
+      id,
       type,
       account_id,
       account_alias,
       control_program,
       transaction_id,
       position,
-      output_id,
```

Comment:
I think we should keep `output_` prefix here because we need to disambiguate between `output_id`, `asset_id` and `account_id` in this form.